src/components/tabs/README.md

```diff
@@ -28,3 +28,11 @@ A basic tab group would have the following markup.
 | Name | Type | Description |
 | --- | --- | --- |
 | `selectedIndex` | `number` | The index of the currently active tab. |
+| `focusIndex` | `number` | The index of the currently active tab. |
+
+### Methods
```

Comment:
Whoops, meant to make that Events.

On Tue, Jun 7, 2016 at 11:22 AM, Jeremy Elbourn <notifications@github.com>
wrote:

> In src/components/tabs/README.md
> <https://github.com/angular/material2/pull/649#discussion_r66125375>:
>
> > @@ -28,3 +28,11 @@ A basic tab group would have the following markup.
> >  | Name | Type | Description |
> >  | --- | --- | --- |
> >  | `selectedIndex` | `number` | The index of the currently active tab. |
> > +| `focusIndex` | `number` | The index of the currently active tab. |
> > +
> > +### Methods
>
> Events?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/material2/pull/649/files/7549b51e9edf89821776897ae85264ad4cf0db01#r66125375>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAi34_rjGxHdf5nxxJbxg2ikFfhOFLAgks5qJbbIgaJpZM4IwM32>
> .
>